src/bin/frontendlauncher

```diff
@@ -130,6 +130,21 @@ do
     fi
 done
 
+# Handle possible language spec
+have_lang=no
+for arg in $@
+do
+    if test "$have_lang" = "yes"; then
+        lang=$arg
+        export LANG="$lang"
+        shift 2
```

Comment:
Why the "shift 2"?

Reply:
To remove the language related args from being passed on to other VisIt components.

Reply:
My interpretation of that loop is that arg loops over the values in $@. At the end of the loop there is an implied shift, When you have a shift, you skip to the next argument, so doing a shift 2 skips 2 arguments in that loop. So what you are doing is skipping 2 arguments from processing in that loop. Since the loop is only looking for "-lang", I think that the only time this would impact things is if you hang "-lang" twice in the list of arguments, one right after the other (e.g. "-lang french -lang english"), which I don't expect anyone would ever do. The case where it would make a difference is if you were processing lots of arguments in that loop, in which case 2 arguments could get unintentionally skipped.

Reply:
Ok, I've adjusted this look  to simply `break` out after exporting `LANG` variable. Works fine.